Source/Initialization/InjectorMomentum.H

```diff
@@ -220,6 +220,49 @@ private:
     int m_flux_direction;
 };
 
+
+// struct whose getMomentum returns momentum for 1 particle, from random
+// waterbag distribution (i.e. a uniform distribution u_min < u < u_max)
+struct InjectorMomentumWaterbag
+{
+    InjectorMomentumWaterbag (amrex::Real a_ux_min, amrex::Real a_uy_min,
+                              amrex::Real a_uz_min, amrex::Real a_ux_max,
+                              amrex::Real a_uy_max, amrex::Real a_uz_max) noexcept
+        : m_ux_min(a_ux_min), m_uy_min(a_uy_min), m_uz_min(a_uz_min),
+          m_ux_max(a_ux_max), m_uy_max(a_uy_max), m_uz_max(a_uz_max)
+        {
+            m_Dux = m_ux_max - m_ux_min;
+            m_Duy = m_uy_max - m_uy_min;
+            m_Duz = m_uz_max - m_uz_min;
+            m_ux_h = 0.5 * (m_ux_max - m_ux_min);
```

Comment:
This should presumably be "+" to take the average.

Reply:
Thanks, when I copy-pasted the lines I missed the typo!